app/Http/Controllers/ExportController.php

```diff
@@ -3,8 +3,7 @@
 namespace App\Http\Controllers;
 
 use App\{Guild, Item};
-use Illuminate\Support\Facades\DB;
-use Illuminate\Support\Facades\Cache;
+use Illuminate\Support\Facades\{App, Cache, DB};

```

Comment:
idk why I didn't think to add `App` to the `use` statements at the top... I guess I had just been working with the blade templates and didn't realize I was back in a controller and could easily do that again lol